src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx

```diff
@@ -47,15 +47,18 @@ const CurrencyRow: React.FC<Props> = ({ lendingPool, lendingAmount }) => {
     );
   };
 
+  const ChartSection = () =>
```

Comment:
Well, this doesn't work as intended. This is what I see on 1244px and it shouldn't be there.

![image](https://user-images.githubusercontent.com/14878713/120491365-7721c500-c3b9-11eb-94b3-4ccb2d36dff3.png)

Also, it's not a good idea to read screen width from window, just create a styled component and use a media query in it (with `display: none`).